csharp.test/TestMaps.cs

```diff
@@ -49,6 +49,15 @@ public static void CanRoundtripRequiredMaps()
             DoRoundtripTest(false, keys, values);
         }
 
+        [Test]
+        public static void CanRoundtripLogicalMapType()
```

Comment:
The logical map type is used in other test cases too but only on the top-level schema node, so this name is a little confusing. Maybe something like:
```suggestion
        public static void CanRoundtripNonStandardMapAnnotation()
```